contracts/Agreement.sol

```diff
@@ -101,4 +105,14 @@ contract Agreement is AccessControl, Ownable {
     function leaveCommunity() public virtual {
       renounceRole("OG", msg.sender);
     }
+
+    function activationSwitch() public onlyOwner returns (string memory) {
+      if (contractActive) {
+        contractActive = false;
+        return "Contract has been Deactivated";
+      } else {
+          contractActive = true;
+          return "Contract has been Activated";
+      }
```

Comment:
this whole function can be refactored to:
```
contractStatus = !contractStatus
```
the return value does not seem necessary since we can simply check the contractStatus state variable